doc/tutorials/setup_logical_partition_scenario_on_kbl_nuc_i7.rst

```diff
@@ -0,0 +1,154 @@
+  Set Maximum number of VCPUs per VM as "2";
+  Set Maximum number of PCPU as "4";
+  Set Serial IO type as "Legacy";
+  Set Base address of serial PIO region as "0x3f8";
+  Set 2M-aligned Start physical address of the RAM region used by the hypervisor as "0x6a000000";
+  Select "Enable ACPI DMA Remapping tables parsing".
+
+Other items is better to keep the default value.
+
+Step 3: Check or update board specific configuration.
+Double check GRUB loader name whether it is provided in the ``hypervisor/arch/x86/configs/nuc7i7bnh/board.c``. If no, should update it::
+
+  update-grub -v
+  grub-mkconfig (GRUB) 2.02~beta2-36ubuntu3.18
+
+In this case, the GRUB loader name shall have ``GRUB`` prefix;
+
+Step 4: Check or update BDF information of PCI devices of each pre-launched VM;
+Check it in the ``hypervisor/arch/x86/configs/nuc7i7bnh/pci_devices.h``.
+
+Step 5: Build the ACRN hypervisor and copy the artifact ``acrn.32.out`` to the ``/boot`` directory::
+
+  $make BOARD=nuc7i7bnh
+
+Note: If you update hypervisor image (``acrn.32.out``) on ubuntu of KBL NUC i7, need use scp to copy it from the remote host building machine.
+If you install EBtool Docker on the ubuntu of KBL NUC i7, it will be easy to update hypervisor image.
+
+Update Ubuntu GRUB to Boot hypervisor and Load Kernel Image
+***********************************************************
+
+Step 1: Append the following configuration to the ``/etc/grub.d/40_custom`` file::
+
+  menuentry 'ACRN hypervisor Logical Partition Scenario' --class ubuntu --class gnu-linux --class gnu --class os $menuentry_id_option 'gnulinux-simple-e23c76ae-b06d-4a6e-ad42-46b8eedfd7d3' {
+          recordfail
+          load_video
+          gfxmode $linux_gfx_mode
+          insmod gzio
+          insmod part_gpt
+          insmod ext2
+
+          echo 'Loading hypervisor logical partition scenario ...'
+          multiboot --quirk-modules-after-kernel /boot/acrn.32.out
+          module /boot/bzImage
+  }
+
+Step 2: Modify the ``/etc/default/grub`` file as follows to make the GRUB menu visible when booting::
+
+  #GRUB_HIDDEN_TIMEOUT=0
+  GRUB_HIDDEN_TIMEOUT_QUIET=false
+
+Step 3: Update grub::
+
+  sudo update-grub
+
+Reboot the KBL NUC i7 board. Select the ACRN hypervisor Logical Partition Scenario entry to boot the logical partition of the ACRN hypervisor on the KBL NUC i7 displayer, GRUB loader will boot the hypervisor, and the hypervisor will start two pre-launched VMs automatically.
+
+Logical Partition Scenario Startup Checking
+*******************************************
+
+Step 1: Check whether ACRN hypervisor startup fails or not, this is similar to SDC scenario on KBL NUC.
+
+Step2: Check whether two pre-launched VMs startup fails or not (such as enter user OS shell command line, devices operations testing).
+Note: Enter pre_launched VM from hypervisor through the hypervisor shell command ``vm_console vm_id`` and find vm_id through hypervisor shell command ``vm_list``.
```

Comment:
Can you add a pointer to the ACRN Shell user guide (using :ref:`acrnshell`)

Reply:
added at doc v2 version